runt.toml

```diff
@@ -577,3 +577,17 @@ make --silent -B TEST_PATH={} COCOTB_LOG_LEVEL=CRITICAL |\
   rm -f results.xml
 
 """
+
+[[tests]]
+name = "Kabir matrix multiplier test"
+paths = ["tests/kabir_practice/matrix_multiplier.futil"]
+cmd = """
+name=$(basename {} .py) &&
```

Comment:
I think you wanted
```
name=$(basename {} .futil) &&
```
That will convert `tests/kabir_practice/matrix_multiplier.futil` into `matrix_multiplier` and store it in `name`. Right now you're trying to nip off a `.py` extension, which is quietly failing, so `name` is getting `matrix_multiplier.futil`. That is why you are being forced to have a `.data` file called `matrix_multiplier.futil.data` instead of `matrix_multiplier.data`, which is what you probably want.

Reply:
I think that's in line with what you were telling me yesterday! I'll change that, thanks!